src/main/java/core/basesyntax/Machine.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public abstract class Machine {
+
```

Comment:
<img width="1013" alt="Screenshot 2022-01-17 at 08 58 18" src="https://user-images.githubusercontent.com/33026471/149723774-50a18072-f25e-475b-8805-92d99ff15ad8.png">
let's remove redundant empty lines here and in main